ledger-core/CMakeLists.txt

```diff
@@ -31,8 +31,33 @@ set(CMAKE_MACOSX_RPATH 1)
 
 add_definitions("-DSQLITE_HAS_CODEC")
 
+set(INSTALL_TARGETS
+    ledger-core
+    ledger-core-static
+    ledger-core-obj
+    ledger-core-interface
+    bigd
+    fmt
+    crypto
+    soci_sqlite3
+    soci_postgresql
+    soci_core_static
+    leveldb
+    spdlog
+    sqlcipher
+    ethash
+    ledger-test
+    ledger-qt-host
+    ledger-core-integration-test
+    mongoose
+    ssl
+    gtest gtest_main
+)
+
 if (PG_SUPPORT)
     add_definitions("-DPG_SUPPORT")
+else()
```

Comment:
I’m not sure this is the right way to do. That seems odd.

Reply:
I would prefer adding to the list in the `if (PG_SUPPORT)` branch too.